docs/devcontainer/viz-tutorial.ipynb

```diff
@@ -0,0 +1,540 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Intro\n",
+    "\n",
+    "This assumes you have gone through the intro/mpi4py intro, and that you are comfortable generating ExaGO output.\n",
+    "\n",
+    "Now we will go through specifics to the vizualisation frontend, how it is installed, and a typical workflow.\n",
+    "\n",
+    "# Open Issues:\n",
+    "\n",
+    "- npm dependencies were pinned and have security issues\n",
+    "- requirements.txt has pinned versions and we cannot upgrade / reproduce\n",
+    "- OpenAI keys and other missing steps...\n",
+    "\n",
+    "# Generating file to visualize\n",
+    "\n",
+    "I don't know how to do gic file option, so we are going to use OPFLOW C++ CLI to do this..."
```

Comment:
what is gic? please define

Reply:
@abhyshr has to clarify this. I don't know about gic file stuff and so I just used the pre-generated data and this is a comment/dox